app/models/episode_import.rb

```diff
@@ -1,208 +1,50 @@
-require "addressable/uri"
-require "feedjira"
-require "itunes_category_validator"
-require "loofah"
-require "hash_serializer"
-
 class EpisodeImport < ApplicationRecord
   include ImportUtils
 
-  serialize :entry, HashSerializer
-  serialize :audio, HashSerializer
+  store :config, coder: JSON
```

Comment:
I've migrated both `podcast_imports` and `episode_imports` table to have this "config" [store field](https://api.rubyonrails.org/classes/ActiveRecord/Store.html).  It's still JSON-encoded, but a bit more modern than the HashSerializer.

In general, these base classes are now pretty minimal.  The diff is large here, but this code all migrated to `app/models/imports/episode_rss_import`.